Reflection/Constants.simba

```diff
@@ -430,3 +437,292 @@ const //Widgets
+  R_SPELL_SNARE: RWidget =                           [R_INTERFACE_SPELLBOOK, 35];
+  R_SPELL_MAGIC_DART: RWidget =                      [R_INTERFACE_SPELLBOOK, 36];
+  R_SPELL_ARDOUGNE_TELEPORT: RWidget =               [R_INTERFACE_SPELLBOOK, 37];
+  R_SPELL_EARTH_BLAST: RWidget =                     [R_INTERFACE_SPELLBOOK, 38];
+  R_SPELL_HIGH_LEVEL_ALCHEMY: RWidget =              [R_INTERFACE_SPELLBOOK, 39];
+  R_SPELL_CHARGE_WATER_ORB: RWidget =                [R_INTERFACE_SPELLBOOK, 40];
+  R_SPELL_LVL_4_ENCHANT: RWidget =                   [R_INTERFACE_SPELLBOOK, 41];
+  R_SPELL_WATCHTOWER_TELEPORT: RWidget =             [R_INTERFACE_SPELLBOOK, 42];
+  R_SPELL_FIRE_BLAST: RWidget =                      [R_INTERFACE_SPELLBOOK, 43];
+  R_SPELL_CHARGE_EARTH_ORB: RWidget =                [R_INTERFACE_SPELLBOOK, 44];
+  R_SPELL_BONES_TO_PEACHES: RWidget =                [R_INTERFACE_SPELLBOOK, 45];
+  R_SPELL_SARADOMIN_STRIKE: RWidget =                [R_INTERFACE_SPELLBOOK, 46];
+  R_SPELL_CLAWS_OF_GUTHIX: RWidget =                 [R_INTERFACE_SPELLBOOK, 47];
+  R_SPELL_FLAMES_OF_ZAMORAK: RWidget =               [R_INTERFACE_SPELLBOOK, 48];
+  R_SPELL_TROLLHEIM_TELEPORT: RWidget =              [R_INTERFACE_SPELLBOOK, 49];
+  R_SPELL_WIND_WAVE: RWidget =                       [R_INTERFACE_SPELLBOOK, 50];
+  R_SPELL_CHARGE_FIRE_ORB: RWidget =                 [R_INTERFACE_SPELLBOOK, 51];
+  R_SPELL_TELEPORT_TO_APE_ATOLL: RWidget =           [R_INTERFACE_SPELLBOOK, 52];
+  R_SPELL_WATER_WAVE: RWidget =                      [R_INTERFACE_SPELLBOOK, 53];
+  R_SPELL_CHARGE_AIR_ORB: RWidget =                  [R_INTERFACE_SPELLBOOK, 54];
+  R_SPELL_VULNERABILITY: RWidget =                   [R_INTERFACE_SPELLBOOK, 55];
+  R_SPELL_LVL_5_ENCHANT: RWidget =                   [R_INTERFACE_SPELLBOOK, 56];
+  R_SPELL_TELEPORT_TO_KOUREND: RWidget =             [R_INTERFACE_SPELLBOOK, 57];
+  R_SPELL_EARTH_WAVE: RWidget =                      [R_INTERFACE_SPELLBOOK, 58];
+  R_SPELL_ENFEEBLE: RWidget =                        [R_INTERFACE_SPELLBOOK, 59];
+  R_SPELL_TELEOTHER_LUMBRIDGE: RWidget =             [R_INTERFACE_SPELLBOOK, 60];
+  R_SPELL_FIRE_WAVE: RWidget =                       [R_INTERFACE_SPELLBOOK, 61];
+  R_SPELL_ENTANGLE: RWidget =                        [R_INTERFACE_SPELLBOOK, 62];
+  R_SPELL_STUN: RWidget =                            [R_INTERFACE_SPELLBOOK, 63];
+  R_SPELL_CHARGE: RWidget =                          [R_INTERFACE_SPELLBOOK, 64];
+  R_SPELL_WIND_SURGE: RWidget =                      [R_INTERFACE_SPELLBOOK, 65];
+  R_SPELL_TELEOTHER_FALADOR: RWidget =               [R_INTERFACE_SPELLBOOK, 66];
+  R_SPELL_WATER_SURGE: RWidget =                     [R_INTERFACE_SPELLBOOK, 67];
+  R_SPELL_TELE_BLOCK: RWidget =                      [R_INTERFACE_SPELLBOOK, 68];
+  R_SPELL_BOUNTY_TARGET_TELEPORT: RWidget =          [R_INTERFACE_SPELLBOOK, 69];
+  R_SPELL_LVL_6_ENCHANT: RWidget =                   [R_INTERFACE_SPELLBOOK, 60];
```

Comment:
Is it a typo? 60 is already defined as `R_SPELL_TELEOTHER_LUMBRIDGE`

Reply:
Yep. Should be 70.